blackbox_test.go

```diff
@@ -0,0 +1,15 @@
+/*
+This file contains complete black box tests of the agent, avoiding any cycle reference
+*/
+
+package scopeagent_test
+
+import (
+	"testing"
+
+	_ "go.undefinedlabs.com/scopeagent/autoinstrument"
+)
+
+func TestComplete(t *testing.T) {
```

Comment:
What is the purpose of this test?

Reply:
Yeah that’s in another PR, i have to remove this file.

Reply:
removed